english/tell-mother/select-option/select-option.md

```diff
@@ -0,0 +1 @@
+By letting your curiosity get the best of you, you selected the new option and wiped your phone clean, losing all data you had stored.  I'm sorry.
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.